### PR TITLE
test: fix line endings in tests on Windows

### PR DESCRIPTION
**Pull request by mortenpi**
I think the issue with the Windows CI is that the `git checkout` checks the reference files out with `\r\n` line endings. But we can just normalize the line endings to `\n` before we string-compare the generated output and the reference output.

### CODE REVIEW

File: src/MultiDocumenter.jl [hunk 1]
```diff
@@ -266,7 +286,8 @@ function maybe_clone(docs::Vector{MultiDocRef})
             catch e
                 # We're only interested in catching `git` errors here
                 isa(e, ProcessFailedException) || rethrow()
-                @error "Unable to update existing clone at $(doc.upstream)" exception = (e, catch_backtrace())
+                @error "Unable to update existing clone at $(doc.upstream)" exception =
+                    (e, catch_backtrace())
```
Comment (mortenpi) [inline src/MultiDocumenter.jl:290]: Unrelated formatting fixing from #61.

File: src/MultiDocumenter.jl [hunk 2]
```diff
@@ -140,7 +140,26 @@ function make(
...
+        else
+            msg = """
+            MultiDocumenter deployments are disabled on Windows due to difficulties
+            with handling symlinks in documentation sources.
+            You _can_ test this build locally by running it interactively (i.e. in the REPL).
+            See also: https://github.com/JuliaComputing/MultiDocumenter.jl/issues/70
+            """
+            error(msg)
+        end
+    end
```
Comment (mortenpi) [inline src/MultiDocumenter.jl:162]: @pfitzseb So initially I thought we could check if we run into problematic symlinks, and only then error. But I don't think that works, because by default Git on Windows will not create symlinks as far as I can tell. So the "symlinks" are just normal files, and so we can't really distinguish them easily. So instead I added the `isinteractive()` checks here at the very top of `make.jl`.

One consequence here is that we fully "break" running MultiDocumenter on Windows. So we may want to bump this to the next 0.x minor?

**Review by pfitzseb — APPROVED** [review]
